packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -73,7 +83,30 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	 * Array of {@link StarkColumnProperties} objects which define the columns of the data table.
 	 */
 	@Input()
-	public columnProperties: StarkTableColumnProperties[] = [];
+	public set columnProperties(input: StarkTableColumnProperties[]) {
+		this._columnProperties = (input || []).map((properties: StarkTableColumnProperties) => ({
+			...DEFAULT_COLUMN_PROPERTIES,
+			...this._columnProperties,
+			label: properties.name, // default See StarkTableColumnProperties
```

Comment:
This specific part is also handled in `column.component.ts:215`... so maybe we should keep it there only. What do you think?